### PR TITLE
Relative imp

### PR DESCRIPTION
**Pull request by nataliecrowe060**
Relative importance function, loop, graphs and summaries. I also cleaned up the markdown a little bit, let me know what you think of the code included/hidden- I left most of it in there.

### CODE REVIEW

File: Methane_MarkDown.Rmd [hunk 1]
```diff
@@ -347,14 +346,140 @@ expsen_data<- sensitivity_emission%>%
...
+  labs(title= "Relitive Importance Co2 Concentration")
+
+#RF Total
+Rel_RF_totlong<- Rel_RF_tot %>%
+  pivot_longer(!c(year, variable), names_to = "parameter", values_to = "value")
+
+Rel_RF_graph<- 
+  ggplot(Rel_RF_totlong, aes(x= year, y= value, fill= parameter))+
+  geom_area()+
+  labs(title= "Relitive Importance RF Total")
```
Comment (bpbond) [inline Methane_MarkDown.Rmd:434]: Here and elsewhere: "Relative"

**Review by bpbond — APPROVED** [review]
Looks good overall @nataliecrowe060 — nice! Could you post (here, or on Slack) what your stacked-area graphs look like?

File: Methane_MarkDown.Rmd (hunk 1 above)
Comment (nataliecrowe060) [reply Methane_MarkDown.Rmd:434]: image

Comment (nataliecrowe060) [reply Methane_MarkDown.Rmd:434]: image

File: Methane_MarkDown.Rmd [hunk 2]
```diff
@@ -347,14 +346,140 @@ expsen_data<- sensitivity_emission%>%
...
+
+Rel_global_tasgraph<-  ggplot(Rel_global_taslong, aes(x= year, y= value, fill= parameter))+
+  geom_area()+
+  labs(title= "Relitive Importance Global Tas")
+
+Rel_cconc_graph
+Rel_RF_graph
+Rel_RF_CO2graph
+Rel_global_tasgraph
+```
```
Comment (leeyap) [inline Methane_MarkDown.Rmd:457]: Graphs look *great* 👍

File: Methane_MarkDown.Rmd (hunk 1 above)
Comment (nataliecrowe060) [reply Methane_MarkDown.Rmd:434]: image

Comment (bpbond) [reply Methane_MarkDown.Rmd:434]: Niiiiiice!

**Review by leeyap — APPROVED** [review]
Great job, as per usual!

**Pull request merged**